### ISSUE TITLE
Run_squad AssertionError

### ISSUE THREAD

**Issue by anil1055** [kamalkraj/BERT-SQuAD#16]
Hi, 
I want to train bert model again with updated dataset. I updated run_squad code according to the train/readme file, but I got AssertionError. How do I resolve this error for train? 

def main():
    parser = argparse.ArgumentParser()
    parser.add_argument("--train_file /train-v1.1.json", default=None, type=str, required=True)
    parser.add_argument("--predict_file /dev-v1.1.json", default=None, type=str, required=True)
    parser.add_argument("--model_type bert ", default=None, type=str, required=True)
    parser.add_argument("--model_name_or_path bert-large-uncased-whole-word-masking", default=None, type=str, required=True)
    parser.add_argument("--output_dir ../models/wwm_uncased_finetuned_squad/", default=None, type=str, required=True)    parser.add_argument("--max_seq_length", default=384, type=int)
    parser.add_argument("--doc_stride", default=128, type=int)
    parser.add_argument("--do_train", action='store_true')
    parser.add_argument("--do_eval", action='store_true')
    parser.add_argument("--do_lower_case", action='store_true')
    parser.add_argument("--per_gpu_train_batch_size", default=3, type=int)
    parser.add_argument("--per_gpu_eval_batch_size", default=3, type=int)
    parser.add_argument("--learning_rate", default=3e-5, type=float)
    parser.add_argument("--num_train_epochs", default=2.0, type=float)
    parser.add_argument("--nproc_per_node", default=8, type=int)
    args = parser.parse_args()

![ErrorFile](https://user-images.githubusercontent.com/17703776/112754038-ff16d780-8fe2-11eb-86eb-dd01314f2206.jpg)
